app/grandchallenge/reader_studies/templates/reader_studies/readerstudy_progress.html

```diff
@@ -49,6 +49,7 @@ <h2>Users Progress for {{ reader_study.title }}</h2>
                                 &nbsp;{{ user.progress.questions|floatformat }}%
                             </div>
                         </div>
+                values: {csrfmiddlewaretoken: target.data("csrf")},
```

Comment:
What is this doing?

Reply:
Probably an accidental paste. It's gone now